tests/FormaterTests.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+import os
+from app.Notebook import Cell, Notebook
+from app.Formater import ToNotebook, ToPy
+
+
+class FormaterTests(unittest.TestCase):
+    def test_topy_class_creation(self):
+        topy = ToPy()
+        self.assertIsInstance(topy, ToPy)
+
+    def test_to_notebook_class_creation(self):
+        tonb = ToNotebook()
+        self.assertIsInstance(tonb, ToNotebook)
+
+    def test_topy_empty_constructor_creation(self):
+        topy = ToPy()
+        self.assertEquals((topy.dry_run, topy.overwrite), (False, False))
+
+    def test_to_notebook_empty_constructor_creation(self):
+        tonb = ToNotebook()
+        self.assertEquals((tonb.dry_run, tonb.overwrite), (False, False))
+
+    def test_nb_constructor(self):
+        tonb = ToNotebook(True, True)
+        self.assertEquals((tonb.dry_run, tonb.overwrite), (True, True))
+
```

Comment:
These seem like testing the implementation instead of behavior. Not the variables should be assured on but rather the change in behavior because of the different values.